app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.spendmanager.demo"
+        android:theme="@style/Theme.Sensibill"
```

Comment:
I wonder if we should do it this way

We could do this and just make the whole application theme exactly our theme, but as this is a demo, it might be closer to clients' use cases if we keep the application theme `Theme.spendmanager.demo` (though maybe we should fix the capitalization on that), but only have the activities we need set to use the sensibill theme.
Additionally maybe we could add a little customization, change a colour or two in the sb theme

Reply:
Adjusted to use sensibill for the web integration methods, and just its own default theme for everything else